case/infrasim/virtual_node/T130052_idic_ESXiTest.py

```diff
@@ -24,6 +24,7 @@ def config(self):
     def test(self):
         gevent.joinall([gevent.spawn(self.boot_to_disk, obj_node)
                        for obj_node in self.stack.walk_node()])
+        time.sleep(10)
```

Comment:
is it for following "arp" command to get qemu ip address to be successful?
If so, should we retry that arp command? It can make code more readable.

Reply:
It is for the failing at get_instance_name in esxcli_test() after power cycle node in boot_to_disk().
Infrasim need time to start after node powercycle.